apis/swagger.yml

```diff
@@ -655,11 +655,23 @@ paths:
         This endpoint is mainly for reporting the heart beat to supernode.
         And supernode could know if peer is alive in strem mode.
       parameters:
-        - name: "body"
-          in: "body"
-          description: "request body which contains peer info."
-          schema:
-            $ref: "#/definitions/HeartBeatRequest"
+        - name: ip
+          in: query
+          required: true
+          description: "IP of peer."
+          type: "string"
```

Comment:
format should be `ipv4`

Reply:
In api defintion, paramter.types should in [string number boolean integer array], which could not set ipv4.

Reply:
`format: 'ipv4'` if we only support `ipv4`

Reply:
done.